draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,18 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint MAY elect to only send or retransmit RETIRE_CONNECTION_ID frames
```

Comment:
```suggestion
An endpoint MAY elect to only send RETIRE_CONNECTION_ID frames
```

Reply:
I really wanted to make it clear that an endpoint might have sent RCID but choose to abandon retransmission if lost, based on this criterion. I may not have worded it well, but I don't think your suggestion fixes it.

Reply:
We typically don't talk about retransmissions, so this read oddly to me.

Reply:
Would it be better say just "send", and then have a separate sentence like "an endpoint MAY abandon retransmission of frames retiring sequence numbers that meet these conditions."?

Reply:
your new text solves this problem, thanks